py/torch_tensorrt/fx/converters/convolution.py

```diff
@@ -1,212 +1,123 @@
-
-        kernel = np.expand_dims(kernel, -1)
-        kernel_size = kernel.shape[2:]
-        if bias is not None:
-            bias = bias[None]
-        stride = (stride[0], 1)
-        padding = (padding[0], 0)
-        dilation = (dilation[0], 1)
-    layer = network.add_convolution_nd(
-        input=input_val,
-        num_output_maps=mod.out_channels,
-        kernel_shape=kernel_size,
-        kernel=kernel,
-        bias=bias,
-    )
-    layer.name = layer_name
-    layer.stride_nd = stride
-    layer.padding_nd = padding
-    layer.dilation_nd = dilation
-    layer.num_groups = mod.groups
-
-    if is_quantized:
-        # Assume the dtype of activation is torch.quint8
-        mark_as_int8_layer(
-            layer, get_dyn_range(mod.scale, mod.zero_point, torch.quint8)
-        )
-
-    result = layer.get_output(0)
-    if dimension == 1:
-        # Append squeeze after conv2d to calculate conv1d
-        squeeze_layer = network.add_shuffle(input=result)
-        squeeze_layer.reshape_dims = tuple(result.shape[:-1])
-        squeeze_layer.name = f"{layer_name}_squeeze"
-        result = squeeze_layer.get_output(0)
-
-    return result
-
-
-def common_conv_relu(network, mod, dimension, input_val, layer_name, is_quantized):
-    conv_output = common_conv(
-        network,
-        mod,
-        dimension=2,
-        input_val=input_val,
-        layer_name=f"{layer_name}_conv",
-        is_quantized=is_quantized,
-    )
-
-    layer = network.add_activation(input=conv_output, type=trt.ActivationType.RELU)
-    layer.name = f"{layer_name}_relu"
-
-    if is_quantized:
-        mark_as_int8_layer(layer, conv_output.dynamic_range)
-
-    return layer.get_output(0)
+logger = logging.getLogger(__name__)
 
 
 @tensorrt_converter(torch.nn.modules.conv.Conv1d)
 def conv1d(network, submod, args, kwargs, layer_name):
     # args/kwargs should have already been normalized to kwargs
     assert len(args) == 0
-    input_val = kwargs["input"]
-
-    if not isinstance(input_val, trt.tensorrt.ITensor):
-        raise RuntimeError(
-            f"Conv1d received input {input_val} that is not part "
-            "of the TensorRT region!"
-        )
 
     if layer_name is None:
         raise RuntimeError("layer name is none")
-    return common_conv(
+    return convolution.convNd(
         network,
-        submod,
-        dimension=1,
-        input_val=input_val,
-        layer_name=layer_name,
-        is_quantized=False,
+        submod._get_name(),
+        source_ir=SourceIR.NN,
+        name=layer_name,
+        is_conv1d=True,
+        input_val=kwargs["input"],
+        weight=submod.weight,
+        bias=submod.bias,
+        stride=getattr(submod, "stride"),
+        padding=getattr(submod, "padding"),
+        dilation=getattr(submod, "dilation"),
+        groups=submod.groups,
     )
 
 
 @tensorrt_converter(torch.nn.modules.conv.Conv2d)
 def conv2d(network, submod, args, kwargs, layer_name):
```

Comment:
Same here, we should get rid of this file and just put the registrations for each opset in the same file